src/components/Playlist/Playlist.jsx

```diff
@@ -0,0 +1,29 @@
+import React, {useEffect} from "react";
+import {PlaylistProducts} from "../../context/CartProvider";
+import {NormalCard} from "..";
+
+export function Cart() {
+  const {PlaylistVideos, removeFromCart} =
+    PlaylistProducts();
+
+  useEffect(() => {});
```

Comment:
You can remove this empty useEffect

Reply:
Okay, I'll do that, Thank you Vineet for PR review.